### DISCUSSION TITLE
How to add a link for preprint

### DISCUSSION THREAD

**Discussion by xinyindeed** [alshedivat/al-folio#2731] [Q&A]
I tried to add the html link for a preprint in papers.bib but the error says "Failed to parse BibTeX on value "html" (NAME)" How can I add a link to preprint on my publication page?

**Comment by riasat-sheikh** [comment]
Could you share a bit more information about it? Maybe the bib entry itself?

**Reply by xinyindeed** [reply]
Actually fixed thank you! I had a typo before
